src/Akeneo/Component/StorageUtils/Cursor/CursorFactoryInterface.php

```diff
@@ -14,10 +14,10 @@
     /**
      * Create the cursor with the correct implementation
      *
-     * @param mixed    $queryBuilder Query builder
```

Comment:
I think you can revert this one

our "ES query" is called "search query builder"

not perfect, but at the moment we don't know if it will be a query or a builder :/